docs/config.rst

```diff
@@ -993,4 +1024,3 @@ Valid preset color variables are:
    :value: $GREEN
 
    Color used for indicating that a display is connected.
-
```

Comment:
It looks like your editor removed this line. Could you please add it back? The Sphinx parser gets upset if the file ends with an indented block.